src/__experimental__/components/radio-button/radio-button.spec.js

```diff
@@ -41,15 +42,26 @@ describe("RadioButton", () => {
     it("does not allow a children prop", () => {
       spyOn(console, "error");
       render({ children: "someChildren" });
-      const expected =
```

Comment:
Why did you change this?

Reply:
Because the message was changed for `ForwardRef`. I assumed that would be sufficient. Would you like the full message there?
![Zrzut ekranu 2021-06-4 o 11 22 29](https://user-images.githubusercontent.com/77274590/120779278-32b43780-c527-11eb-9bf6-7b89fcfecd5d.png)

Reply:
Could we not set `RadionButton.displayName = "RadioButton"` here to get the same error message output as before?

Reply:
Actually you are right, good point thanks. Changed.